roblib/islandora_fits
# Work on #1411.

**GitHub Issue**: https://github.com/Islandora/documentation/issues/1411

# What does this Pull Request do?

Adds an additional field for FITS data, the PUID field populated by Droid.

# What's new?

This PR parses out the value of the `<externalIdentifier toolname="Droid" toolversion="6.4" type="puid">fmt/12</externalIdentifier>` element of the FITS XML, and then adds a new "Droid" member to the array of metadata already extracted from the FITS XML.

# How should this be tested?

1. check out this PR
1. add a new node and accompanying media
1. look for the following at the end of the FITS metadata output:

![Capture](https://user-images.githubusercontent.com/403918/73326882-9b3f2600-4208-11ea-876a-8df5864d23d8.PNG)

(Value with vary depending on the format of the media.)


# Interested parties
@ajstanley, @Islandora/8-x-committers
